gotopkg.go

```diff
@@ -1,204 +0,0 @@
-package main
-
-import (
-	"flag"
-	"fmt"
-	"github.com/groob/plist"
```

Comment:
Hmm.. I'll consider it - I have a slight allergy to tacking even more tools inline to the save process in the editor. I would have thought if it were important enough the standard `go fmt` would take care of it?